### DISCUSSION TITLE
Compliance Trestle Release Planning

### DISCUSSION THREAD

**Discussion by jpower432** [oscal-compass/compliance-trestle#1535] [Ideas]
Hello `compliance-trestle` community,

I am adding this discussion to be a place where we can collaboratively discuss strategies around release planning and roadmap planning for `compliance-trestle`.

## Objectives

- Choose a method to document what issue/task will go into a particular release
- Add a document in the repository to help contributors understand the release planning process

## Proposed Options

- Use milestones to track releases
- Assign issues to milestones
- Use GitHub Projects to track the status of each issue and release completion using a Backlog and Kanban view
- Create automation around how issues are managed and moved in the board

I invite/encourage anyone to share any feedback, ideas, or concerns.

**Comment by jpower432** [comment]
@vikas-agarwal76 @degenaro @AleJo2995 @mrgadgil To track some action items around milestones from the community call (April 23rd). If there is anything you would like to sign up for or if there is something you believe is missing, please reply. If you have a preferred way to track these (GitHub Issues?), please let me know.

- [x] Close current sprint based milestone
- [x] Create a new milestone for the next major version and add relevant issues and pull requests
- [x] Research solutions for issue tracking (ZenHub , GitHub Projects, etc...)

**Reply by butler54** [reply]
This makes a lot of sense to me (jumping back into the community a little).

I ran into a few issues around dependencies where trestle is mandating increasingly aged versions which would be best updated when a major release is completed.

**Reply by jpower432** [reply]
Thanks for the reply @butler54! Based on some feedback from our last community meeting, we will be conducting an issue review session to kick things off. I will post details once scheduled.
